pandas/core/generic.py

```diff
@@ -7247,12 +7247,15 @@ def interpolate(
               given length of interval.
             * 'index', 'values': use the actual numerical values of the index.
             * 'pad': Fill in NaNs using existing values.
-            * 'nearest', 'zero', 'slinear', 'quadratic', 'cubic', 'spline',
+            * 'nearest', 'zero', 'slinear', 'quadratic', 'cubic',
               'barycentric', 'polynomial': Passed to
-              `scipy.interpolate.interp1d`. These methods use the numerical
+              `scipy.interpolate.interp1d`, whereas 'spline' is assed to
+              `scipy.interpolate.UnivariateSpline`. These methods use the numerical
               values of the index.  Both 'polynomial' and 'spline' require that
               you also specify an `order` (int), e.g.
-              ``df.interpolate(method='polynomial', order=5)``.
+              ``df.interpolate(method='polynomial', order=5)``. Note that, Scipy
+              `slinear` method refers to the Scipy first order spline instead
+              of Pandas first order spline.
```

Comment:
> Note that, Scipy `slinear` method refers to the Scipy first order spline instead of Pandas first order spline.

Feels like this is explaining scipy, but should explain pandas. Maybe

> Note that `slinear` in pandas is different than `slinear` in Scipy and results in pandas may differ from scipy.

?

Reply:
Exactly, good catch! All typos fixed.